logstash-output-OMS.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.metadata = { "logstash_plugin" => "true", "logstash_group" => "output" }
 
   # Gem dependencies
-  s.add_runtime_dependency "logstash-core", ">= 2.0.0", "< 3.0.0"
+  s.add_runtime_dependency "logstash-core", ">= 1.4.0", "< 2.0.0"
```

Comment:
I see - a single output plugin for both version skus would be awesome, but second branch can be used for now